src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/RemoveRepresentationConfirmationTest.java

```diff
@@ -52,18 +51,8 @@ void should_apply_noc_and_return_confirmation() {
             removeRepresentationConfirmation.handle(callback);
```

Comment:
Test should include for event REMOVE_LEGAL_REPRESENTATIVE
Suggest using @ParameterizedTest for event

Reply:
Done